tests/src/androidTest/java/com/squareup/duktape/DuktapeGetTest.java

```diff
@@ -330,4 +331,50 @@
       assertThat(expected).hasMessage("Unsupported Java type class java.util.Date");
     }
   }
+
+  interface TestVarArgs {
+    String call(String separator, Object... args);
+  }
+
+  @Test public void callVarArgMethod() {
+    duktape.evaluate(""
+        + "function joiner() {\n"
+        + "  var result = '';\n"
+        + "  for (i = 0; i < arguments.length; i++) {\n"
+        + "    if (i > 0) result = result + this;\n"
+        + "    result = result + arguments[i].toString();\n"
+        + "  }\n"
+        + "  return result.toString();\n"
+        + "}");
+    TestVarArgs joiner = duktape.get("joiner", TestVarArgs.class);
+    assertThat(joiner.call("-")).isEqualTo("");
+    assertThat(joiner.call(" + ", 1, 2, "three")).isEqualTo("1 + 2 + three");
+
+    try {
+      joiner.call(", ", "Test", new Date(), 1.0);
+      fail();
+    } catch (Exception expected) {
+      assertThat(expected).hasMessage("Unsupported Java type class java.util.Date");
+    }
+  }
+
+  interface Summer {
+    double sum(double... args);
```

Comment:
wanna do the boilerplate of repeating with booleans and ints? You’re already duplicating code in the implementation so it’s prudent to cover it all